core/server/worker/src/main/java/alluxio/worker/page/PagedBlockReader.java

```diff
@@ -164,6 +168,11 @@ public ReadableByteChannel getChannel() {
   @Override
   public int transferTo(ByteBuf buf) throws IOException {
     Preconditions.checkState(!mClosed);
+    // todo(bowen): mUfsBlockOptions may not always carry blockSize. get correct size from
+    // block metastore
```

Comment:
@beinan Please do not merge this until #16157 is merged, so I can address this based on changes in that PR.

Reply:
just merged #16157